codec-http/src/main/java/io/netty/handler/codec/http/cookie/DefaultCookie.java

```diff
@@ -119,6 +120,14 @@ public void setHttpOnly(boolean httpOnly) {
         this.httpOnly = httpOnly;
     }
 
+    public String sameSite() {
```

Comment:
@dvlato please add some java docs

Reply:
Hi @normanmaurer happy to add javadocs but still wondering if you want a String here or an enum, as @slandelle mentioned (and I had considered myself as per my comment that this solution might not be accept for you)